README.md

```diff
@@ -32,3 +32,17 @@ Destroy isn't always possible from Terraform. Manual steps found to be:
 Details on managing the user credentials can be found [here](https://git.ucd.gpn.gov.uk/dip/aws-common-infrastructure/wiki/Manual-CI-Credential-Rotation#github-actions-user)
 
 This user will deactivate its own credentials after every use.  In order to activate them again, you must run [this job](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/concourse-admin?group=credentials).
+
+
+## AMI testing
+
+Upon a new releae of Concourse, or an update to the Concourse AMI config, we build a new AMI named: `untested-dw-al2-concourse-ami` using the [ami-builder](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder) pipeline in Concourse.  Upon successful creation of a new [untested-dw-al2-concourse-ami](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/resources/untested-dw-al2-concourse-ami) resource, we will trigger the [validate-concourse-ami](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/jobs/validate-concourse-ami) job.  The only way this job will swucceed is if the test results provided by the [concourse](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/resources/concourse) resource, match the AMI ID provided by the [untested-dw-al2-concourse-ami](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/resources/untested-dw-al2-concourse-ami) resource, and contain the word `SUCCESS`.
+
+The [concourse](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/resources/concourse) resource is provided in an non-standard pattern, which differs entirely from all other test result resources in the [ami-builder](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder) pipeline.  This is mainly due to Concourse being deplpoyed via `GitHub Actions` and not Concourse, unlike every other consumer of our AMIs.
```

Comment:
`The [concourse](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder/resources/concourse) resource is provided in an non-standard pattern, which differs entirely from all other test result resources in the [ami-builder](https://ci.dataworks.dwp.gov.uk/teams/dataworks/pipelines/ami-builder) pipeline.  This is mainly due to Concourse being deployed via `GitHub Actions` and not Concourse, unlike every other consumer of our AMIs.`